01_fib/jimbotech/ex-fibonacci.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// fibonacci returns a function that returns
+// an number in the fibonacci sequence.
+// This is an implmentation of "closure".
+// The inside function returns the current number
+// in the fibonnacci sequence, starting at 1
+// and sets up the next which will be returned
+// in the subsequent call
+func fibonacci() func() int {
```

Comment:
The tab takes up 8 character spaces here. I would think normally a tab/indentation would be 2 or 4 spaces.

Reply:
I agree, I much prefer 4 characters, but after having spent a lot of time beating my editor into submission, I found "go fmt" simply puts the 8 space tab right back. I also checked other peoples submissions and they all use 8 space tabs. As much as I am agreeing with you, I have been defeated and will not be able to implement your suggestion.

Reply:
I think you are right, the editor shows up as normal indentation, and gofmt puts it back as 8.